README.md

```diff
@@ -533,6 +533,9 @@ arXiv:2307.10593, 2023. [PDF](https://arxiv.org/pdf/2307.10593.pdf), [Video](htt
 *[DOTIE - Detecting Objects through Temporal Isolation of Events using a Spiking Architecture](https://ieeexplore.ieee.org/abstract/document/10161164).*,<br/>
 IEEE Int. Conf. Robotics and Automation (ICRA), 2023. [Arxiv](https://arxiv.org/abs/2210.00975), [CVPR 2023 workshop](https://tub-rip.github.io/eventvision2023/papers/2023CVPRW_Live_Demonstration_Real-time_Event-based_Speed_Detection_using_Spiking_Neural_Networks.pdf), [Code](https://github.com/manishnagaraj/DOTIE).
 - [Gao et al., ICCV 2023](#Gao23iccv), *A 5-Point Minimal Solver for Event Camera Relative Motion Estimation*.
+- <a name="Li24cvpr"></a>Li, S., Zhou, Z., Xue, Z., Li, Y., Du, S., Gao, Y.,<br/>
+  *3D Feature Tracking via Event Camera*,<br/>
```

Comment:
Is there a pdf for this?

Reply:
Thanks! PDF will be available in CVPR proceedings. We will update the link when it is online.